README.md

```diff
@@ -1,24 +1,67 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails Budget App
 
-Things you may want to cover:
+> Rails Budget App is an app build on Ruby on Rails. It is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+## Demo Video
 
-* System dependencies
+[Video Link](https://www.loom.com/share/ade2cd157e2946918b7f409d9047d5fa)
 
-* Configuration
+## Live Demo Link
 
-* Database creation
+[Live Link](https://rails-budget-app-white3d.herokuapp.com/)
 
-* Database initialization
+## Built With
 
-* How to run the test suite
+![Ruby](https://icongr.am/devicon/ruby-original.svg?size=50&color=currentColor)
+![Ruby on Rails](https://icongr.am/devicon/rails-original-wordmark.svg?size=50&color=currentColor)
+![Ruby](https://icongr.am/devicon/postgresql-original.svg?size=50&color=currentColor)
 
-* Services (job queues, cache servers, search engines, etc.)
+## Get Started
 
-* Deployment instructions
+### Run
 
-* ...
+```
+To get a clone of the project, run `git clone https://github.com/white3d/Rails-Budget-App.git`
+```
+
+```
+cd /Rails-Budget-App
+```
+
+
+### Available Scripts
+
+Gets all the required dependencies of the projects.
+
+**`bundle install`**
+
+For more details, see [the Installation docs page](https://www.ruby-lang.org/en/).
+
+## Authors
+
+👤 **Edward Odhiambo**
+
+- GitHub: [@whit3d](https://github.com/white3d)
+- Twitter: [@odhiambo_ed](https://twitter.com/odhiambo_ed)
+- LinkedIn: [Edward Odhiambo](https://www.linkedin.com/in/edward-odhiambo-6a462a21b/)
+- Portfolio: [Edward Odhiambo](https://edwardodhiambo.com/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!!!
+
+Feel free to check the [issues page](https://github.com/white3d/Rails-Budget-App/issues)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the original design owner.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Like the previous reviewer suggested, the link to the MIT license is broken. Please fix it because it is advisable for every software (opensource or not) to have a license. 👍